METRICS.md

```diff
@@ -36,8 +38,9 @@ The path `rtt` is the round trip time histogram for each request/response
 message pair.
 
 The `tx` and `rx` metrics count the number of message of a given type
-transmitted and received. The `timeout` counter exists only for requests
-that require a response.
+transmitted and received. The `rr` metric count the number of message processed
```

Comment:
@mgumz I don't like the ```rr``` short name. Should we use ```lb``` (load balancer) or ```ld``` (load director) instead?

Reply:
@surik what does the "rr" stand for, actually?

Reply:
@mgumz  `rr` - redirected requests. It should be something like `rrx` to have the same naming with `rx` and `tx` but because we only have two letters it becomes `rr` :)

Reply:
why is there a "rr" metric at all? if gtphub runs in redirector mode, all the rx and tx metrics are related only to that mode, right?

Reply:
Yes, we can use these common `rr` and `tx` after a small fix in metrics code. For now I just remove `rr`.